pkgs/development/python-modules/west/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k
+, colorama, configobj, packaging, pyyaml, pykwalify
+, pytest
+}:
+
+buildPythonPackage rec {
+  version = "0.7.2";
+  pname = "west";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "11dbzlcg48fymddqjrrs60pr7y33qjjv0y5zrfjc56gkc190gmz6";
+  };
+
+  propagatedBuildInputs = [
+    colorama
+    configobj
+    packaging
+    pyyaml
+    pykwalify
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
Good point, thank you. Fixed and force-pushed.